docs/mkdocs/docs/problems.md

```diff
@@ -0,0 +1,198 @@
+        File "/workspaces/partcad.worktrees/devel/.venv/lib/python3.10/site-packages/google/api_core/gapic_v1/method.py", line 131, in __call__
+          return wrapped_func(*args, **kwargs)
+        File "/workspaces/partcad.worktrees/devel/.venv/lib/python3.10/site-packages/google/api_core/retry/retry_unary.py", line 293, in retry_wrapped_func
+          return retry_target(
+        File "/workspaces/partcad.worktrees/devel/.venv/lib/python3.10/site-packages/google/api_core/retry/retry_unary.py", line 153, in retry_target
+          _retry_error_helper(
+        File "/workspaces/partcad.worktrees/devel/.venv/lib/python3.10/site-packages/google/api_core/retry/retry_base.py", line 212, in _retry_error_helper
+          raise final_exc from source_exc
+        File "/workspaces/partcad.worktrees/devel/.venv/lib/python3.10/site-packages/google/api_core/retry/retry_unary.py", line 144, in retry_target
+          result = target()
+        File "/workspaces/partcad.worktrees/devel/.venv/lib/python3.10/site-packages/google/api_core/timeout.py", line 120, in func_with_timeout
+          return func(*args, **kwargs)
+        File "/workspaces/partcad.worktrees/devel/.venv/lib/python3.10/site-packages/google/api_core/grpc_helpers.py", line 78, in error_remapped_callable
+          raise exceptions.from_grpc_error(exc) from exc
+      google.api_core.exceptions.ResourceExhausted: 429 Resource has been exhausted (e.g. check quota).
+      INFO:  Visualizing in "OCP CAD Viewer"...
+      WARN: Unable to open config file /tmp/sandbox/home/partcad-cli-j5iagz3t/.ocpvscode.
+      WARN: No VS Code or "OCP CAD Viewer" extension detected.
+      INFO:  Script candidate 0: successfully rendered the image
+      INFO:  Visualizing in "OCP CAD Viewer"...
+      [ 2 / 20 ] inspect this; 162s
+        [Show] /:generated-case [162s]
+      WARN: Unable to open config file /tmp/sandbox/home/partcad-cli-j5iagz3t/.ocpvscode.
+      WARN: No VS Code or "OCP CAD Viewer" extension detected.
+      INFO:  Script candidate 0: successfully rendered the image
+      INFO:  So far 2 valid script candidates
+      INFO:  Attempting to select the best script by comparing images
+      INFO:  Selected the best fit image: 2
+      INFO:  Visualizing in "OCP CAD Viewer"...
+      [ 1 / 23 ] inspect this; 168s
+        [Show] /:generated-case [168s]
+      WARN: Unable to open config file /tmp/sandbox/home/partcad-cli-j5iagz3t/.ocpvscode.
+      WARN: No VS Code or "OCP CAD Viewer" extension detected.
+      INFO:  DONE: inspect: this: 168.58s
+      Aborted.
+
+      DEBUG:root:Command error: WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
+      E0000 00:00:1735443942.784201    9830 init.cc:229] grpc_wait_for_shutdown_with_timeout() timed out.
+
+      DEBUG:root:Command return code: 1
+```
+
+1. Do not pollute output with traces.
+   - In order to not loose details (to file a ticket) verbose log could be saved to file for convenience.
```

Comment:
1. This is standard and expected API error `429 Resource has been exhausted (e.g. check quota).` and there is no need to show trace in that case.

Reply:
Maybe user should see something like this?

```diff
-ERROR: 429 Resource has been exhausted (e.g. check quota).  
-Traceback (most recent call last):  
-  File "/workspaces/partcad.worktrees/devel/partcad/src/partcad/ai_google.py"...  
+ERROR: Resource quota exceeded (PC-AI-429). See logs at ~/.partcad/logs/latest.log  
```